apsi-enrollment-be/src/main/resources/db/changelog/db.changelog-0.3.xml

```diff
@@ -0,0 +1,26 @@
+
```

Comment:
In this project we use one changelog per release. I would suggest moving those changes to 0.2

Reply:
moved